blog/en/blog/2022/07/29/release-apache-apisix-2.15.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Release Apache APISIX 2.15"
+authors:
+  - name: "Zexuan Luo"
+    title: "Author"
+    url: "https://github.com/spacewander"
+    image_url: "https://github.com/spacewander.png"
+  - name: "Sylvia"
+    title: "Technical Writer"
+    url: "https://github.com/SylviaBABY"
+    image_url: "https://avatars.githubusercontent.com/u/39793568?v=4"
+keywords: 
+- Apache APISIX
+- Release
+- Custom plugin
+- Plugin
+- Flexibility
+description: Apache APISIX 2.15 is officially released! This release brings some new features at the plugin level to provide more flexibility when using plugins.
+tags: [Community]
+---
+
+> Apache APISIX 2.15 is officially released! This release brings some new features at the plugin level to provide more flexibility when using plugins.
+
+<!--truncate-->
+
+Since the first 2.0 version was released two years ago, APISIX has released 15 minor versions and many patch versions. As the last minor version of the 2.x series, version 2.15 can be said to be a link between the previous and the next.
+
+"Link" is because this version continues to introduce more functions, making plugin configuration more flexible, and also will be the last LTS version of version 2.x. Later APISIX will be moving to version 3.0 in the future. Let's see below for more details!
+
+![Update summary](https://static.apiseven.com/2022/blog/0729/blog-en1.png)
+
+### Custom plugin priority
+
+The new version allows users to customize the priority of the plugin instead of directly applying the plugin's default priority properties.
+
+With this feature, we can adjust the execution order of several plugins on a specific route, thus breaking the constraints of the previous plugin priority properties.
+
+By default, the `serverless-post-function` plugin is executed after the `serverless-pre-function` plugin. But with this feature, the `serverless-post-function` plugin can be made to execute first. A configuration example is as follows:
+
+```json
+ {
+    "serverless-post-function": {
+        "_meta": {
+            "priority": 10000
+        },
+        "phase": "rewrite",
+        "functions" : ["return function(conf, ctx)
+                    ngx.say(\"serverless-post-function\");
+                    end"]
+    },
+    "serverless-pre-function": {
+        "_meta": {
+            "priority": -2000
+        },
+        "phase": "rewrite",
+        "functions": ["return function(conf, ctx)
+                    ngx.say(\"serverless-pre-function\");
+                    end"]
+    }
+}
+```
+
+If a plugin configuration does not indicate priority, it is sorted by the priority property value in the plugin code.
+If you specify the priority of the plugin in the plugin configuration of the Service or Plugin Config, it will still take effect after merging into the route.
```

Comment:
```suggestion

If you specify the priority of the plugin in the plugin configuration of the Service or Plugin Config, it will still take effect after merging into the route.
```

Reply:
Need a new line between two paragraphs.